src/azure-cli-core/setup.py

```diff
@@ -55,6 +55,7 @@
     'applicationinsights',
     'argcomplete>=1.8.0',
     'colorama',
+    'humanfriendly',
     'jmespath',
     'msrest>=0.4.4',
     'msrestazure>=0.4.7',
```

Comment:
Changes in this file shouldn't be required for this feature.
Are they for progress?